metadata.json

```diff
@@ -49,6 +49,9 @@
         "6",
         "7"
       ]
+    },
+    {
+      "operatingsystem": "Archlinux"
```

Comment:
Don't add this here as this will make our modulesync template to add an acceptance test entry in the .travis.yml file but we do not have a nodeset for it yet (maybe you could provide one?).